boards/shields/mikroe_wifi_bt_click/doc/index.rst

```diff
@@ -0,0 +1,77 @@
+.. _mikroe_wifi_bt_click:
+
+MikroElektronika WIFI and BLE Shield
+####################################
+
+Overview
+********
+
+This shield contains an Espressif Systems ESP32 which is responsible
+to offload the WIFI and Bluetooth tasks from the host processor. It uses
+the standard MikroBus interface with UART pins plus its Espressif AT
+command set. Also Zephyr modem driver could be used in place, to used
+this module as a native WIFI networking interface
+
+.. image:: ./wifi_click.jpeg
+   :width: 634px
+   :align: center
+   :alt: MikroE WIFI BLE Click
+
+Pins Assignment of the Shield
+=============================
+
++-----------------------+---------------------------------------------+
+| Shield Connector Pin  | Function                                    |
++=======================+=============================================+
+| RST#                  | ESP32 Reset                                 |
```

Comment:
We should add a note that there is a conflict with the rst polarity and we need to disconnect the RST pin on most board (the LPC55S69 at least).